script/popup.js

```diff
@@ -106,9 +106,12 @@ function main() {
         li.addEventListener('mouseleave', function (event) {
             event.target.classList.remove('item--hover');
         });
+        li.addEventListener('click', function () {
+            chrome.tabs.create({active: true, url: link.href})
```

Comment:
a要素にclickイベントを登録するときは、a要素とonclickの挙動が2重にならないように`event.preventDefault()`があると嬉しいです 👍 
```suggestion
        li.addEventListener('click', function (event) {
            event.preventDefault();
            chrome.tabs.create({active: true, url: link.href});
```

Reply:
Suggestいただいたものをそのままコミットしました 🙏 db827909fe7078160b8d410d72ab952779685c4e